src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -96,23 +130,34 @@ public LocalDateTime getDateInSpecificTimeZone(String dateInString, String zone)
      * OffsetDateTime советуют использовать при записи даты в базу данных.
      */
     public OffsetDateTime offsetDateTime(LocalDateTime localTime) {
-        return OffsetDateTime.now();
+        return OffsetDateTime.of(localTime, ZoneOffset.of("+02:00"));
     }
 
     /**
      * Дана строка в виде "yyyymmdd".
      * Необходимо вернуть Optional даты в LocalDate формате
      */
     public Optional<LocalDate> parseDate(String date) {
-        return Optional.empty();
+        try {
+            DateTimeFormatter forma = DateTimeFormatter
+                    .ofPattern("yyyyMMdd");
+            return Optional.of(LocalDate.parse(date, forma));
```

Comment:
```suggestion
            return Optional.of(LocalDate.parse(date, DateTimeFormatter.BASIC_ISO_DATE));
```
you don't need to create your own formatter here